certified-connectors/Polaris PSA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2613 @@
+                                "properties": {
+                                  "year": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "year of task start date"
+                                  },
+                                  "month": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "month of task start date"
+                                  },
+                                  "day": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "day of task start date"
+                                  }
+                                },
+                                "description": "task strat date"
+                              }
+                            },
+                            "description": "timeEntryStartDateToApply"
+                          },
+                          "timeEntryEndDateToApply": {
+                            "type": "object",
+                            "properties": {
+                              "date": {
+                                "type": "object",
+                                "properties": {
+                                  "year": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "year of task end date"
+                                  },
+                                  "month": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "year of task end date"
+                                  },
+                                  "day": {
+                                    "type": "integer",
+                                    "format": "int32",
+                                    "description": "year of task end date"
+                                  }
+                                },
+                                "description": "task end date"
+                              }
+                            },
+                            "description": "timeEntryEndDateToApply"
+                          },
+                          "timeAndExpenseEntryTypeToApply": {
+                            "type": "object",
+                            "properties": {
+                              "value": {
+                                "type": "string",
+                                "description": "time and expense entry type value"
+                              }
+                            },
+                            "description": "timeAndExpenseEntryTypeToApply"
```

Comment:
@repl-anand-vikram There are many description values in the swagger need to be fixed, please update them